draft-ietf-taps-interface.md

```diff
@@ -833,6 +834,14 @@ selection is necessarily tied to path selection. This may involve choosing
 between multiple local interfaces that are connected to different access
 networks.
 
+When additional information (such as Provisioning Domain (PvD) information 
+{{RFC7556}}) is provided about the networks over which an endpoint can operate,
+this can inform the selection between alternate network paths.  
```

Comment:
This seems like the same issue that @abrunstrom pointed out about a different part of this text: "is provided" gives me the impression that it might be the application's job to provide this information to the transport system via the API. That's not what this is, right? In this case, I'd rather go with something like:

When the transport system has additional information (..). ... available ... about the networks over which ...

Reply:
available will do. I will rework.

Reply:
I think you've done this but not resolved this conversation. Clicking the button.